ucm2/MediaTek/mt8390-evk/HiFi.conf

```diff
@@ -0,0 +1,168 @@
+SectionDevice."HDMI1" {
+	Comment "HDMI output"
+
+	Value {
+		PlaybackPriority 300
+		PlaybackPCM "hw:${CardId},5"
+		JackControl "HDMI Jack"
+	}
+
+	EnableSequence [
+		cset "name='HDMI_OUT_MUX' 1"
+	]
+
+	DisableSequence [
+		cset "name='HDMI_OUT_MUX' 0"
+	]
+}
+
+SectionDevice."HDMI2" {
+	Comment "DP output"
+
+	Value {
+		PlaybackPriority 300
+		PlaybackPCM "hw:${CardId},5"
+		JackControl "DP Jack"
+	}
+
+	EnableSequence [
+		cset "name='DPTX_OUT_MUX' 1"
+	]
+
+	DisableSequence [
+		cset "name='DPTX_OUT_MUX' 0"
+	]
+}
+
+SectionDevice."Speaker" {
+	Comment "Lineout speaker"
+
+	ConflictingDevice [
+		"Headphones"
+	]
+
+	EnableSequence [
+		cset "name='LOL Mux' Playback_L_DAC"
+		cset "name='Lineout Volume' 10"
+	]
+
+	DisableSequence [
+		cset "name='LOL Mux' Open"
+	]
+
+	Value {
+		PlaybackPriority 400
+		PlaybackChannels 2
+		PlaybackPCM "hw:${CardId},0"
+	}
+}
+
+SectionDevice."Headphones" {
+	Comment "Earphone speaker"
+
+	ConflictingDevice [
+		"Speaker"
+	]
+
+	EnableSequence [
+		cset "name='HP Mux' Audio Playback"
+		cset "name='Headset Volume' 2"
+	]
+
+	DisableSequence [
+		cset "name='HP Mux' Open"
+	]
+
+	Value {
+		PlaybackPriority 500
+		PlaybackChannels 2
+		PlaybackPCM "hw:${CardId},0"
+	}
+}
+
+SectionDevice."Headset" {
+	Comment "Earphone microphone"
+
+	ConflictingDevice [
+		"Mic1"
+	]
+
+	EnableSequence [
+		cset "name='PGA_L_Mux' AIN1"
+	]
+
+	Value {
+		CapturePriority 500
+		CaptureChannels 1
+		CapturePCM "hw:${CardId},10"
+	}
+}
+
+SectionDevice."Mic1" {
+	Comment "Analog microphone"
+
+	ConflictingDevice [
+		"Headset"
+	]
+
+	EnableSequence [
+		cset "name='PGA_L_Mux' AIN0"
+		cset "name='PGA1 Volume' 4"
```

Comment:
Volume presets should be in BootSequence.

Reply:
Done